src/components/Diagram/Diagram.js

```diff
@@ -20,15 +30,30 @@ const Diagram = () => {
             .style('cursor', 'pointer');
     }, []);
 
-    return (
+    return (  
         <Box
-            alignItems='center'
             className={classes.root}
             display='flex'
             flexDirection='row'
+            justifyContent = 'space-between'
+            alignItems='stretch'
         >
-            <SVGDiagram />
-        </Box>
+            <Box
+                className={classes.inner}
+                alignItems='left'
+                alignContent='stretch'
+                width = {0.4}
```

Comment:
We should let layout handle the width of the boxes and not define width ourselves

Reply:
Both boxes should have equal amount of width. We can specify the width to be 100% for both boxes and specify an equal minWidth both of them can have.